gem/management/commands/molo_makemessages.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
@Mitso lets not commit this now since it's not a generic script, we need to refactor this a bit so we would be able to give it different values for `fromDirectory`, `locale` and `ignore`